package.json

```diff
@@ -8,7 +8,7 @@
     "prestart": "bash git-version",
     "start": "ng serve",
     "prebuild": "bash git-version",
-    "build": "ng build --prod --build-optimizer --vendor-chunk=true",
+    "build": "ng build --configuration production --build-optimizer --vendor-chunk=true",
```

Comment:
--build-optimizer --vendor-chunk=true are need here? or these shall go with aot etc, in config file for production.

Reply:
We can remove  --vendor-chunk as that is for dev purpose only as per angular doc 
> --vendor-chunk | Generate a separate bundle containing only vendor libraries. This option should only used for development.

Reply:
Read more https://angular.io/cli/build